mlflow/server/js/src/experiment-tracking/components/App.js

```diff
@@ -76,7 +78,7 @@ class App extends Component {
                     <span>GitHub</span>
                   </div>
                 </a>
-                <a href={'https://mlflow.org/docs/latest/index.html'}>
+                <a href={HomePageDocsUrl}>
```

Comment:
This change relates to https://github.com/mlflow/mlflow/pull/5577#pullrequestreview-931321957